vllm/worker/habana_model_runner.py

```diff
@@ -183,6 +184,75 @@ def align_workers(value, op):
     return value_t.item()
 
 
+def setup_profiler():
```

Comment:
This can be removed, since its overridden in 245

Reply:
Removed